### PR TITLE
Pkg 2020 tests

### PR DESCRIPTION
**Pull request by markan**
Turn tests back on to validate with jupyter_server

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,6 +31,13 @@ requirements:
     - terminado >=0.8.3
     - pywinpty >=2.0.10 # [win]
 
+{% set suppressed_tests = "" %}
+{% set min_cov = 80 %}
+
+# Some tests fail in CI, apparently due to quirks around running from cygwin, not from pwsh or cmd
+{% set suppressed_tests = "-k \"not (test_authorized_requests or test_terminal_create or test_terminal_create_with_kwargs or test_terminal_create_with_cwd)\"" %} # [win]
+{% set min_cov = 50 %} # [win ]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
{% set min_cov = 50 %}  # [win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -29,6 +31,13 @@ requirements:
     - terminado >=0.8.3
     - pywinpty >=2.0.10 # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - pywinpty >=2.0.10  # [win]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -29,6 +31,13 @@ requirements:
...
 
+{% set suppressed_tests = "" %}
+{% set min_cov = 80 %}
+
+# Some tests fail in CI, apparently due to quirks around running from cygwin, not from pwsh or cmd
+{% set suppressed_tests = "-k \"not (test_authorized_requests or test_terminal_create or test_terminal_create_with_kwargs or test_terminal_create_with_cwd)\"" %} # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
{% set suppressed_tests = "-k \"not (test_authorized_requests or test_terminal_create or test_terminal_create_with_kwargs or test_terminal_create_with_cwd)\"" %}  # [win]
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -39,10 +48,12 @@ test:
     - jupyter_server_terminals  # needs juypyter_server
   commands:
     - pip check
-    - pytest -vv --cov=jupyter_server_terminals --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under=80
+    - pytest -vv {{ suppressed_tests }} --cov=jupyter_server_terminals --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under={{ min_cov }}
   requires:
     - jupyter_server >=2.0.0rc8
+    - m2-bash # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - m2-bash  # [win]
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -39,10 +48,12 @@ test:
...
     - pip
+    - posix # [win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - posix  # [win]
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but two spaces before selector `#`

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**